index.html

```diff
@@ -161,32 +169,37 @@ <h2 class="title cards">Multi-Post Stories</h2>
             </li>
         </ul>
 
-        <a href="#" class="button project">See Project</a>
+        <a href="#" class="button project" id="last">See project</a>
     </div>
+</div>
     </section>
 
     <section id="myself">
 
-        <h2 class="title">
+        <div>
+            <h2 class="title" id="last_h2">
                 About Myself
             </h2>
-            <p class="text">
+            <p class="text" id="last_p">
                 I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
             </p>
-            <p id="color">LET’S CONNECT</p>
-
-            <ul class="social" id="social">
+            <p id="color" class="last">LET’S CONNECT</p>
+
+            <div id="final">
+                <a href="#"><img src="images/twitter.svg" id="twitter" class="icon"></a>
+                <a href="#"><img src="images/linkedin.svg"id="linkedin" class="icon"></a>
+                <a href="#"><img src="images/hand.svg" id="hand" class="icon"></a>
+                <a href="#"><img src="images/git.svg" id="github" class="icon"></a>
+                <a href="#"><img src="images/M.svg" id="m" class="icon"></a>
+             </ul></div>
+            <a id="resume">Get my resume</a>
+        </div>
```

Comment:
you did well implementing the changes by the previous reviewer, however it seems you reduced the padding of the buttons alot . please increase it a bit 😄 

Figma design :
![image](https://github.com/migcm06/miguel-portfolio/assets/20537712/62799d79-b2fa-4ae0-a883-58f6ebe5418a)


Your design :
![image](https://github.com/migcm06/miguel-portfolio/assets/20537712/133c3705-288f-4601-9368-65c3b5a2878b)